go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/tailscale/tailsql
 
-go 1.24.0
+go 1.24.4
+
+toolchain go1.24.5
```

Comment:
Hm, I should probably tweak the script to drop this. But I can do that separately.